subprojects/core/src/integTest/groovy/org/gradle/execution/taskgraph/ParallelTaskExecutionIntegrationTest.groovy

```diff
@@ -146,6 +147,7 @@ class ParallelTaskExecutionIntegrationTest extends AbstractIntegrationSpec imple
         }
     }
 
+    @ToBeFixedForIsolatedProjects(because = "Allprojects in setup")
```

Comment:
💭  When there is a violation in the setup, would it make sense to put the annotation also on the test class level?

We don't have to change it here, since the work has already been done, but it may be helpful for the annotating other Gradle modules.

Reply:
Now ToBeFixedWithIsolatedProjects is not supposing for type application. It'll require some improvements in the spec extension, so I would plan and do that in a separate PR

Reply:
https://github.com/orgs/gradle/projects/63/views/10?pane=issue&itemId=63545824